generator/src/main/java/com/linkedin/pegasus/generator/TemplateSpecGenerator.java

```diff
@@ -131,9 +132,18 @@ public void registerDefinedSchema(DataSchema schema)
    * Generate {@link ClassTemplateSpec} from the specified {@link DataSchema} and its location.
    */
   public ClassTemplateSpec generate(DataSchema schema, DataSchemaLocation location)
+  {
+    return generate(schema, location, false);
+  }
+
+  /**
+   * Generate {@link ClassTemplateSpec} from the specified {@link DataSchema} and its location. If skipDeprecatedField
+   * is set to true, the class of the deprecated field will be bypassed in the generated specs.
```

Comment:
Change this to a proper `@param` Javadoc explanation, and please explain that the effect is recursive.

Reply:
updated